src/InfluxDB/Point.php

```diff
@@ -122,6 +122,15 @@ public function getTags()
      */
     public function setTags($tags)
     {
+        foreach ($tags as &$tag) {
+            if ($tag === '') {
```

Comment:
@sergeyklay this case already covered by `Point::addSlashes`, added test for this case